ceph/ceph_admin/bootstrap_mixin.py

```diff
@@ -0,0 +1,69 @@
+"""
+Cephadm Bootstrap the ceph cluster
+"""
+
+import logging
+from utility.utils import get_cephci_config
+
+logger = logging.getLogger(__name__)
+
+
+class BootstrapMixin:
+    def bootstrap(self):
+        """
+        Bootstrap the ceph cluster with supported options
+
+        Bootstrap involves,
+          - Creates /etc/ceph directory with permissions
+          - CLI creation with bootstrap options with custom/default image
+          - Execution of bootstrap command
+        """
+        # copy ssh keys to other hosts
+        self.cluster.setup_ssh_keys()
+
+        # set tool download repository
+        self.set_tool_repo()
+
+        # install/download cephadm package on installer
+        self.install_cephadm()
+
+        # Create and set permission to ceph directory
+        self.installer.exec_command(cmd="sudo mkdir -p /etc/ceph")
```

Comment:
Let's opt to use `sudo=True` as an argument to `exec_command`

Reply:
Done